### DISCUSSION TITLE
API Documentation

### DISCUSSION THREAD

**Discussion by ptrxyz** [SVG-Edit/svgedit#890] [General]
I might be stupid and all, but I really tried yet I can't seem to find the documentation for the Editor API.

I would like to set the zoom level to "fit to all content" as soon as the editor opens (after pre-loading an image, but I got that working). Can anyone point me to the documentation? I really can't seem to find it... :/

**Comment by jfhenon** [comment]
ahah ! no you're not stupid (or at least not with your question). There is not yet a good documentation about the API and I would love a brave contributor to start this. If I had to find the API, I would look into the svgedit code to see how it was implemented. 
In this case, the button in the bottom panel to "fit to content" use the following code:
 ```
  switch (value) {
      case 'canvas':
      case 'selection':
      case 'layer':
      case 'content':
        this.editor.zoomChanged(window, value)
```
So you may need to look into zoomChanged to adjust the zoom to fit the content.
I hope this helps.
